assets/js/modules/analytics-4/datastore/properties.js

```diff
@@ -691,6 +733,38 @@ const baseResolvers = {
 			yield fetchGetPropertyStore.actions.fetchGetProperty( propertyID );
 		}
 	},
+	*getPropertyCreateTime() {
+		const registry = yield Data.commonActions.getRegistry();
+
+		const propertyID = registry
+			.select( MODULES_ANALYTICS_4 )
+			.getPropertyID();
+
+		const propertyCreateTime = registry
+			.select( MODULES_ANALYTICS_4 )
+			.getPropertyCreateTime();
+
+		if (
+			propertyCreateTime ||
+			propertyID === PROPERTY_CREATE ||
+			! propertyID
```

Comment:
It seems that util is used for `UA` properties:
```
export function isValidPropertyID( propertyID ) {
	return typeof propertyID === 'string' && /^UA-\d+-\d+$/.test( propertyID );
}
```

It would not fit here. I will see to create additional util for GA4, after I attend to all other feedback, if I can see what pattern ga4 property uses

Reply:
@zutigrm ^